bundles/CoreBundle/Migrations/Version20230720161000.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Model\DataObject;
+
+final class Version20230720161000 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Updates class definition files';
+    }
+
+    public function up(Schema $schema): void
+    {
+        try {
+            $list = new DataObject\ClassDefinition\Listing();
+            foreach ($list->getClasses() as $class) {
+                $this->write(sprintf('Saving class: %s', $class->getName()));
+                $class->save();
+            }
+
+            $list = new DataObject\ClassDefinition\CustomLayout\Listing();
+            foreach ($list->getLayoutDefinitions() as $layout) {
+                $this->write(sprintf('Saving custom layout: %s', $layout->getName()));
+                $layout->save();
+            }
+        } catch (DataObject\Exception\DefinitionWriteException $e) {
+            $this->write(
+                'Could not write class definition file. Please set PIMCORE_CLASS_DEFINITION_WRITABLE env.' . "\n" .
+                sprintf(
+                    'If you already have migrate the definitions you can skip this migration via "php bin/console doctrine:migrations:version --add %s"',
```

Comment:
```suggestion
                    'If you have already migrated the definitions, you can skip this migration via "php bin/console doctrine:migrations:version --add %s"',
```

Reply:
This wording is also in other migrations (Version20210706090823, Version20210107103923, Version20211117173000, Version20230508121105, Version20230516161000). Should we change it everywhere?

Reply:
@blankse Thank you for the PR. The changes look good. Could you please do the wording changes in a follow up PR ?

Reply:
@aryaantony92 Yes, that can I make :)

Reply:
@blankse Thank you!

Reply:
@jdreesen @aryaantony92 I created a new PR for that: https://github.com/pimcore/pimcore/pull/15760